conda/utils.py

```diff
@@ -7,6 +7,36 @@
 from os.path import abspath, isdir, join
 import os
 
+
+def can_open(file):
+    """
+    Return True if the given ``file`` can be opened for writing
+    """
+    try:
+        fp = open(file, "ab")
```

Comment:
Opening in `w` will truncate the file.  I added `b` to ensure no weird side effects (or failed opens) by trying to open a binary file without encoding it as a binary.  Probably not an issue since I never actually read it, so just happen I guess.

Is there a downside to adding `b`?